api/working-with-extensions/continuous-integration.md

```diff
@@ -9,108 +9,58 @@ MetaDescription: Use Continuous Integration for testing Visual Studio Code exten
 
 # Continuous Integration
 
-Extension tests can be run on CI services. The `vscode` npm module provides a built-in command (`bin/test`) which:
-
-1. Downloads and unzips VS Code;
-2. Launches your extension tests inside VS Code;
-3. Prints the results to the console and exits with an appropriate status code.
-
-The command will expose some optional environment variables, which you can use to customize the build:
-
-| Name                      | Description                                                                                    |
-| ------------------------- | ---------------------------------------------------------------------------------------------- |
-| `CODE_VERSION`            | Version of VS Code to run the tests against (e.g. `0.10.10`)                                   |
-| `CODE_DOWNLOAD_URL`       | Full URL of a VS Code drop to use for running tests against                                    |
-| `CODE_TESTS_PATH`         | Location of the tests to execute (default is `process.cwd()/out/test` or `process.cwd()/test`) |
-| `CODE_EXTENSIONS_PATH`    | Location of the extensions to load (default is `process.cwd()`)                                |
-| `CODE_TESTS_WORKSPACE`    | Location of a workspace to open for the test instance (default is CODE_TESTS_PATH)             |
-| `CODE_LOCALE`             | Display language to use when running the tests (default is English)                            |
-| `CODE_DISABLE_EXTENSIONS` | Disable all other extensions except the one that is being tested                               |
-| `CODE_TESTS_DATA_DIR`     | Allows to specify the user-data-dir for the tests to use and thus enables to run multiple tests at the same time |
+Extension tests can be run on CI services. The `vscode-test` repository itself contains a sample extension that is tested on Azure Devops Pipelines. You can check out the [build pipeline](https://dev.azure.com/vscode/VSCode/_build?definitionId=14) or jump directly to the [build definition yaml file](https://github.com/microsoft/vscode-test/blob/master/azure-pipelines.yml).
```

Comment:
The link https://github.com/microsoft/vscode-test/blob/master/azure-pipelines.yml doesn't work.